packages/admin-ui/ui/src/i18n/index.ts

```diff
@@ -16,10 +16,14 @@ export const supportedLanguages = [
     locale: "pt",
     name: "Português (Brasil)",
   },
+  {
+    locale: "ru",
+    name: "Русский",
+  },
 ]
 
 const backendUrl = window.location.origin
-const adminPath = process.env.ADMIN_PATH ? `${process.env.ADMIN_PATH}/` : "/"
+const adminPath = process.env.ADMIN_PATH ? `${process.env.ADMIN_PATH}` : "/"
```

Comment:
This part is outdated, could you resolve the conflicts please? Then we can merge the translation.